Makefile

```diff
@@ -229,15 +230,15 @@ generate-database-dumps:
 test-ci: export CI_BUILD=1
 test-ci:
 	CGO_ENABLED=$(CGO_ENABLED) BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/build.sh'"
-	~/.go/bin/go test ./... -v $(TESTARGS) -timeout 120m
+	~/.go/bin/go test ./... -v $(TESTARGS) -json -cover -timeout 120m | tparse
 
 .PHONY: test-sql
 test-sql:
 	$(MAKE) -C internal/db/sqltest/ test
 
 .PHONY: test
 test:
-	go test ./... -timeout 30m
+	go test ./... -timeout 30m -json -cover | tparse -follow
```

Comment:
I added `-follow` here because if you're running it locally you probably want to see some text output while it's running.